include/dev/RedundantADC.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <EVT/io/ADC.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace RedundantADC {
+
+/**
+ * This class allows processing readings from multiple ADCs and checking for redundancy.
+ */
+class RedundantADC {
+public:
+    /**
+     * @brief Enum class representing the status of RedundantADC processing.
+     */
+    enum class Status {
+        OK = 0,                /**< No error. */
```

Comment:
Enum comments should look like this
```suggestion
         /** No error  */
        OK = 0,
```